addons/common/functions/fnc_setHearingCapability.sqf

```diff
@@ -11,6 +11,8 @@
  * None
  *
  * Public: Yes
+ *
+ * Note: uses player
```

Comment:
Thats not true though. The issue with `player` is that it doesn't work with remote controled units zeus. (And drones in case of `vehicle player`)